extensions/builder.md

```diff
@@ -0,0 +1,145 @@
+# Builder
+
+A **builder** is an image that MUST contain an implementation of the lifecycle, and build-time environment, and MAY contain buildpacks. Platforms SHOULD use builders to ease the build process.
+
+## Table of Contents
+
+## Required File/Directories
+A builder MUST have the following directories/files:
+- `/cnb/order.toml` &rarr; As defined in the [platform specification][order-toml-spec]
+- `/cnb/stack.toml` &rarr; As defined in the [platform specification][stack-toml-spec]
+- `/cnb/lifecycle/<lifecycle binaries>` &rarr; An implementation of the lifecycle, which contains the required lifecycle binaries for [building images][lifecycle-for-build].
+
+In addition, every buildpack blob contained on a builder MUST be stored at the following file path:
+- `/cnb/buildpacks/...<buildpack ID>/<buildpack version>/`
+
+## Environment Variables/Labels
+A builder's environment is the build-time environment of the stack, and as such, MUST fulfill all of the [build image specifications][build-image-specs].
+
+The following variables MUST be set in the builder environment (through the image config's `Env` field):
+
+| Env Variable    | Description
+|-----------------|--------------------------------------
+| `WorkingDir`  | The working directory where buildpacks should default (eg: `/workspace`)
```

Comment:
```suggestion
```
This isn't an environment variable, if we want this, it should go somewhere else. (I am assuming you are referring to WorkingDir as defined in the OCI spec)

Also, I don't think currently set `WorkingDir` on builder images?

Reply:
I'll take this out. I had put this in the initial RFC, but as you noted on the RFC, it shouldn't matter for platform API 0.5+
https://github.com/buildpacks/rfcs/pull/116/commits/08b7a1fecf75d69ad21bb04bd6cf9341aa006ba0#r489609978